src/main/java/ru/yandex/money/gradle/plugins/grafana/dashboard/impl/DashboardSender.java

```diff
@@ -78,8 +78,12 @@ private static String getHttpResponseBodyAsString(HttpResponse response) throws
     }
 
     private String getAuthHeader() {
-        return "Basic " + Base64.getEncoder().encodeToString(
-                (grafanaUploadSettings.getUser() + ':' + grafanaUploadSettings.getPassword()).getBytes(UTF_8));
+        if(grafanaUploadSettings.getApiKey().isEmpty()) {
```

Comment:
@josselinchevalay i think it will be better to invert this if statement. And you should check that apiKey property is not null before. Smth like this:
```java
if (grafanaUploadSettings.getApiKey() != null && !grafanaUploadSettings.getApiKey().isEmpty()) {
  return "Basic " + grafanaUploadSettings.getApiKey();
}
// old way
```

Reply:
@komarovd95 i'm not sure if we no declare Api key we can use Basic auth else we use Bearer and api key

Reply:
see that http://docs.grafana.org/http_api/auth/#create-api-token